src/styles/hc-2018/layout.styl

```diff
@@ -0,0 +1,205 @@
+@import url(https://assets.hackcambridge.com/fonts/PxGroteskRegular-Regular.css);
```

Comment:
I would suggest we load the fonts via a link tag in the sponsorship template as it provides [a performance benefit](https://stackoverflow.com/a/12380004).